src/member.rs

```diff
@@ -418,7 +418,7 @@ where
         if u_round <= round_in_progress + rounds_margin {
             self.store.add_unit(su, false);
         } else {
-            debug!(target: "AlephBFT-member", "{:?} Unit {:?} ignored because of too high round {} when round in progress is {}.", self.index(), full_unit, u_round, round_in_progress);
+            warn!(target: "AlephBFT-member", "{:?} Unit {:?} ignored because of too high round {} when round in progress is {}.", self.index(), full_unit, u_round, round_in_progress);
```

Comment:
This is something that might happen when joining a session late, in which case we would see tons of warnings in logs. Maybe trace is better.

Reply:
I'm not sure, the round margin seems to be quite large usually, so I'd expect joining _this_ late to actually be a symptom of a problem. I think in this case I would like to see it in the logs even on low levels of logging.

I'm not very convinced I'm right though, so if you are unconvinced by the above arguments I'll just change it to `trace!`.

Reply:
Ok, you are right, warn is fine. This should not happen during normal execution.